neural_compressor/torch/algorithms/weight_only/gptq.py

```diff
@@ -758,14 +755,16 @@ def find_params(self, x, weight=False):
         self.maxq = self.maxq.to(dev)
         # NF4 FP4
         if self.wdtype != "int":
-            from .rtn import quant_weight
+            from .utility import quant_tensor
 
-            _, scale, zero = quant_weight(
-                x,
-                self.wbits,
-                self.group_size,
+            tmp = x.clone()  # tmp will be replaced after quant_tensor
+
+            _, scale, zero = quant_tensor(

```

Comment:
How about replacing zero with zero_point?

Reply:
The `zero` is using the same representation as popular GPTQ.